MrtCore/console_unpackaged_app/console_unpackaged_app.cpp

```diff
@@ -22,30 +22,20 @@ HRESULT MddBootstrapInitializeWrapper()
     HRESULT hr{ MddBootstrapInitialize(majorMinorVersion, versionTag, minVersion) };
     if (FAILED(hr))
     {
-        wprintf(L"Error 0x%X in MddBootstrapInitialize(0x%08X, %s, %hu.%hu.%hu.%hu)\n",
+        wprintf(L"Error 0x%08X in MddBootstrapInitialize(0x%08X, %s, %hu.%hu.%hu.%hu)\n",
             hr, majorMinorVersion, versionTag, minVersion.Major, minVersion.Minor, minVersion.Build, minVersion.Revision);
         return hr;
     }
 
     return S_OK;
 }
 
-ResourceManager InitMrtCore()
-{
-    // Required for C++/WinRT. This call associates this thread with an apartment and initializes COM runtime.
-    init_apartment();
-
-    // Create a resource manager using the resources index (PRI file) generated during build.
-    auto factory = winrt::get_activation_factory<ResourceManager, IResourceManagerFactory>();
-    return factory.CreateInstance(L"console_unpackaged_app.pri");
-}
-
 int wmain(int argc, wchar_t* argv[])
 {
     // Print usage help.
-    if ((argc < 2) || (wcscmp(argv[1], L"--help") == 0) || (wcscmp(argv[1], L"-?") == 0))
+    if ((argc < 2) || (_wcsicmp(argv[1], L"--help") == 0) || (_wcsicmp(argv[1], L"-?") == 0))
     {
-        std::cout << "Usage: console_unpackaged_app.exe [options] [mode]\n"
+        std::wcout << "Usage: console_unpackaged_app.exe [options] [mode]\n"

```

Comment:
If this is `wcout`, does the string need to be prefixed with `L""` ?

Reply:
It worked without a L prefix :)